backend/src/main/java/neuefische/capstone/camunda/CamundaController.java

```diff
@@ -0,0 +1,27 @@
+package neuefische.capstone.camunda;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/api/camundaprocesses")
+@RequiredArgsConstructor
+public class CamundaController {
+
+    private final CamundaService service;
+
+    @GetMapping()
+    public ResponseEntity<String> writeCamundaProcessesToDB(){
```

Comment:
Da diese Methode nichts zurückgibt, aber Seiteneffekte hat, schlage ich `POST` als HTTP-Methode vor.
```suggestion
    @PostMapping()
    public ResponseEntity<?> writeCamundaProcessesToDB(){
```